src/java/org/apache/cassandra/net/MessagingService.java

```diff
@@ -210,6 +207,20 @@
     static AcceptVersions accept_messaging = new AcceptVersions(minimum_version, current_version);
     static AcceptVersions accept_streaming = new AcceptVersions(current_version, current_version);
 
+    public enum Version
```

Comment:
@dcapwell It _is_ only used in the tests, so I suppose we could move it there for now?

Reply:
I put it here in hopes it would be maintained; if it stays in the test than new versions are likely not to be added